jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/BuildFiles.java

```diff
@@ -53,33 +69,22 @@ private static BuildFileSpec toBuildFileSpec(
    *
    * @param projectRoot the root context directory of this build
    * @param buildFilePath a file containing the build definition
-   * @param templateParameters a map of templating variables to apply on the file before parsing
+   * @param buildOptions the build configuration from the command line
```

Comment:
maybe we can add a small `e.g ` at the end to make this more clear?

Reply:
I could add en example, but not sure what you're for an example of?

Reply:
Oops I totally missed that buildOptions was of type `JibCli`.  Adding an example is probably not useful here. Should the name be a more general, like `cliOptions`, if JibCli also includes things like verbosity and console?